kudo-operator/docs/latest/submission.md

```diff
@@ -1,2 +1,160 @@
+
+```yaml
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: spark-pi
+  namespace: spark
+spec:
+  type: Scala
+  mode: cluster
+  image: "mesosphere/spark:spark-2.4.3-hadoop-2.9-k8s"
+  imagePullPolicy: Always
+  mainClass: org.apache.spark.examples.SparkPi
+  mainApplicationFile: "local:///opt/spark/examples/jars/spark-examples_2.11-2.4.3.jar"
+  arguments:
+    - "150000"
+  sparkConf:
+    "spark.ui.port": "4041"
+  sparkVersion: "2.4.3"
+  restartPolicy:
+    type: Never
+  driver:
+    cores: 1
+    memory: "512m"
+    labels:
+      version: 2.4.3
+    serviceAccount: spark-driver
+  executor:
+    cores: 1
+    instances: 2
+    memory: "512m"
+    labels:
+      version: 2.4.3
+```
+
+Basically, all the Spark application configuration is placed under `spec` section. Here you can specify Spark related configuration properties, such as number of executors, number of cores for drivers/executors, amount of memory and the other. There is also a `sparkConf` section, where you can place configuration parameters in a form of key-value pairs. In the example we override the default `spark.ui.port` with a custom value.
+
+After the application spec is ready, the following command can be used to run it on the operator:
+```
+$ kubectl create -f ./resources/spark-pi.yaml
+```
+
+To describe the newly created application, you can use:
+```bash
+$ kubectl get sparkapplications.sparkoperator.k8s.io spark-pi -n spark
+```
+Pay attentions to the `-n spark` flag - it the name of the namespace, where our application is deployed. 
+
+To get the list of pods:
+```bash
+$ kubectl get pods -n spark
+NAME                                  READY   STATUS      RESTARTS   AGE
+spark-gq4bf8-spark-867b8c8d79-btt4q   1/1     Running     0          58m
+spark-gq4bf8-spark-init-cdcrn         0/1     Completed   0          58m
+spark-pi-1571911449587-exec-1         1/1     Running     0          4s
+spark-pi-1571911449587-exec-2         1/1     Running     0          4s
+spark-pi-driver                       1/1     Running     0          11s
+```
+From the output above we can see the app has been created and is currently running as three pods: one driver and two executors.
+
+To see pod's logs, use:
+```
+$ kubectl logs -f spark-pi-driver -n spark
+```
+Get the last `n` lines of pod's log:
+```
+$ kubectl logs --tail=n spark-pi-driver -n spark
+```
+You can get the full description of the driver pod with the following command: 
+```
+$ kubectl describe pod spark-pi-driver -n spark
+```
+Verify the driver pod is completed successfully:
+```bash
+$ kubectl logs --tail=20 spark-pi-driver -n spark | grep 'Pi is'
+Pi is roughly 3.141644502283289
+```
+
+### Accessing Spark UI
+When running Spark applications on K8s, it could be useful to have an access to Spark UI to monitor and inspect Spark job executions in a web browser. 
+
+*Tip: *
```

Comment:
shall this be a header or removed completely? I think the explanation fits well as a continuation of "overview" and doesn't need any special treatment